scripts/release.sh

```diff
@@ -12,7 +12,6 @@ if test -z "$latestRcTag"; then
     exit 1
 fi
 echo $latestRcTag
```

Comment:
you can ommit this file from the pull request, there are no relevant changes here